src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -113,11 +113,15 @@
             new String[] {String.valueOf(1)}, null, getClass(), null);
 
     private static String getPath(String filename) {
-        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filename;
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/main/" + filename;
+    }
+
+    private static String getNonPackagePath(String filePath) {
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filePath;
```

Comment:
I mentioned package, not file
` Main.main("-c", "/google_checks.xml", "-t", getPath("checks/metrics"));`
how should I get path with modified version of `getPath` method?